src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx

```diff
@@ -124,9 +125,11 @@ export default function ViewViabilityTestModal(props: ViewViabilityTestModalProp
             );
           })}
         </Grid>
-        <Grid item xs={12}>
-          Chart
-        </Grid>
+        {selectedTest.testResults && selectedTest.testResults.length > 0 ? (
```

Comment:
ternary not needed for these rendering use-cases?

```
{ selectedTest.testResults && selectedTest.testResults.length > 0  && (
   <Grid item xs={12}>
       <ObservationsChart observations={selectedTest.testResults} />
   </Grid>
)}
```

Reply:
Sometimes when I do that, and testResults exists but it has length 0, it renders a 0

Reply:
that's because the first check needs to be
```
{ selectedTest.testResults !== undefined
```
ok ternary is fine if you are more comfortable that way.

Reply:
if you were seeing zero it's most likely because some other check was `array.length` instead of `array.length > 0` since both are truthy checks in javascript. you can experiment when you have time to understand this better.

Reply:
Yes. Thank you for the explanation. It must have been that. Not checking the `> 0 `